common/extended_window.hpp

```diff
@@ -20,8 +20,8 @@ namespace ILLIXR{
         Window                  win;
         GLXContext              glc;
         xlib_gl_extended_window(int _width, int _height, GLXContext _shared_gl_context){
-            width = _width;
-            height = _height;
+            width = 2048;
```

Comment:
You should not be overriding these variables like this. Instead, the function parameters should be set where `xlib_gl_extended_window` is called from, in `runtime_impl`.

Reply:
@mhuzai Sorry for the mistakes. Now I've changed the wrong values in gldemo/plugin.cpp from 1024 each to the exact 2K values which you've provided. I also changed the values which were being called in runtime_impl.hpp from 896x640 to exact 2k values and also corrected the mistakes in timewarp_gl/plugin.cpp

Thanks

Reply:
Thanks for fixing the constants but as mentioned in #132, we would like to define the resolution only in one place: in a header file in the  `common/` directory. Perhaps `data_format`, or maybe a new file for just constants. Then, all the places that you made changes in would simply include that header file and get the resolution values. This would ensure that changing the resolution changes it for all the components.

Reply:
I think the only file in the common/ is extended_window.hpp in which there are things like width and height where i can make changes . I have checked the whole  data_format.hpp but i don't think there is anything to modify that will change the resolution values.

Either I have to create a new file which i will have to include in each and every part of the code or the second option that i am understanding is to make changes in extended_window.hpp.

Kindly help and guide if i am going in correct direction.

Thanks :)

Reply:
Yes, there are currently no defines for resolution in `common/`. But that is what we have to add. New defines for height and width, set to 2k, that can be used in the other components.

Reply:
@mhuzai Okay I got it, I have created a new file common/default_resolution.hpp in which I just initialised and declared width and height variable. Now i guess including it in the files like extended_window , timewarp, runtime will do the task.

Thanks